sql/hive/src/main/scala/org/apache/spark/sql/hive/orc/package.scala

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hive
+
+import org.apache.spark.annotation.Experimental
+import org.apache.spark.sql.{DataFrame, SaveMode}
+
+package object orc {
+  /**
+   * ::Experimental::
+   *
+   * Extra ORC file loading functionality on [[HiveContext]] through implicit conversion.
+   *
+   * @since 1.4.0
+   */
+  @Experimental
+  implicit class OrcContext(sqlContext: HiveContext) {
+    /**
+     * ::Experimental::
+     *
+     * Loads specified Parquet files, returning the result as a [[DataFrame]].
```

Comment:
Oops... Thanks for pointing out the typo, but I just removed this file in favor of the new DataFrame reader/writer API :)